### PR TITLE
refactored test functions into their respective utils folders

### PR DESCRIPTION
**Pull request by dumebi**
Refactored each Dapp test function into their respective `.utils.ts`file

### CODE REVIEW

File: test/helpers/story/drivers/dapp-utils/aave.utils.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+export const aaveLendTest = async (hre: HardhatRuntimeEnvironment): Promise<void> => {
+  const { getNamedSigner } = hre
+  const borrower = await getNamedSigner('borrower')
+  const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
+  const { details } = loan
+  const borrowedAmount = await details.lendingToken.balanceOf(
+    details.borrower.address
+  )
+  let shouldPass = true
+  if (!loan) {
```
Comment (sfg11) [inline test/helpers/story/drivers/dapp-utils/aave.utils.ts:69]: @dumebi what would make this condition fail? if the loan doesn't exist?

File: test/helpers/story/drivers/dapp-utils/aave.utils.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import chai, { expect } from 'chai'
```
Comment (sfg11) [inline test/helpers/story/drivers/dapp-utils/aave.utils.ts:1]: @dumebi why single quotes ('') on some and double quotes ("")on others?

File: test/helpers/story/drivers/dapp-utils/compound.utils.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        details.loan.id,
+        details.lendingToken.address,
+      )
+    const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+
+    const tokenAddresses = await diamond.getEscrowTokens(details.loan.id)
+    tokenAddresses.should.not.include(cToken.address)
+
+    const cDaiBalance = await cToken.balanceOf(escrowAddress)
+    cDaiBalance.eq(0).should.eql(true, '')
```
Comment (sfg11) [inline test/helpers/story/drivers/dapp-utils/compound.utils.ts:71]: @dumebi not sure what the '' is for but maybe add a custom error message so when something fails we be like oooo dis balance should be equal but it ain't... ya know?

File: test/helpers/story/drivers/dapp-utils/pooltogether.utils.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      .poolTogetherWithdrawAll(details.loan.id, details.lendingToken.address)
+
+    const tokenAddresses = await diamond.getEscrowTokens(details.loan.id)
+    tokenAddresses.should.not.include(poolTicket.address)
+    const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+    const daiBalance = await details.lendingToken.balanceOf(escrowAddress)
+    daiBalance.should.be.gt('0')
+  }
+
+export const pooltogetherLendTest = async (hre: HardhatRuntimeEnvironment): Promise<void> => {
```
Comment (sfg11) [inline test/helpers/story/drivers/dapp-utils/pooltogether.utils.ts:57]: man... im sorry this is nit-picky but I cant help but say it.... should be`poolTogetherLendTest` not `pooltogetherLendTest` 😅  right??

File: test/helpers/story/drivers/dapp-utils/yearn.utils.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      .connect(details.borrower.signer)
+      .yearnDeposit(
+        details.loan.id,
+        details.loan.lendingToken,
+        details.loan.borrowedAmount
+      )
+
+    const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+    const daiBalance = await details.lendingToken.balanceOf(escrowAddress)
+    daiBalance.eq(details.loan.borrowedAmount).should.eql(true, '')
```
Comment (sfg11) [inline test/helpers/story/drivers/dapp-utils/yearn.utils.ts:25]: @dumebi could this be easierly read by saying `dailBalance.should.eq(details.loan.borrowedAmount)` (idk if this is what this line is trying to say tbh lol) 😝

File: test/helpers/story/drivers/dapp-utils/aave.utils.ts (hunk 1 above)
Comment (dumebi) [reply test/helpers/story/drivers/dapp-utils/aave.utils.ts:69]: yes

File: test/helpers/story/drivers/dapp-utils/aave.utils.ts (hunk 2 above)
Comment (dumebi) [reply test/helpers/story/drivers/dapp-utils/aave.utils.ts:1]: I had hoped prettier would sort that, as out. thank you for pointing

File: test/helpers/story/drivers/dapp-utils/pooltogether.utils.ts (hunk 4 above)
Comment (dumebi) [reply test/helpers/story/drivers/dapp-utils/pooltogether.utils.ts:57]: I have an issue with this, seeing it's same word. But i guess for readability, you're right

File: test/helpers/story/drivers/dapp-utils/yearn.utils.ts (hunk 5 above)
Comment (dumebi) [reply test/helpers/story/drivers/dapp-utils/yearn.utils.ts:25]: yes, you're right

**Review by passabilities — APPROVED** [review]

**Pull request merged**